### ISSUE TITLE
Feature Wish: Copy a route

### ISSUE THREAD

**Issue by JesseWeinstein** [Zaczero/osm-relatify#24]
It would be great to have a way in the app (I presume JOSM can do it already) to duplicate an existing route so it could be modified, as there are often very similar routes (i.e. a few stops skipped on weekends) that are painful to create in the app now.

**Comment by Zaczero** [comment]
Yup, that's something I want work on for sure. When saving a route, you can alternatively choose to save it as a new relation and automatically connect with a route_master of your choice.

**Comment by ekatyshev** [comment]
Also it would be great to be able load master relation into the editor to work with its child realtion with no need to find their id
